rewrite-java/src/main/java/org/openrewrite/java/format/BinaryAlignWhenMultiline.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.format;
+
+import org.openrewrite.*;
+
+public class BinaryAlignWhenMultiline extends Recipe {
```

Comment:
There should not be a separate recipe for this. Its visitor should be part of `AutoFormat`, responding appropriately to the relevant `Style`

Reply:
but we will need a style detector to detect this